source/agora/utils/TxBuilder.d

```diff
@@ -257,7 +259,7 @@ public struct TxBuilder
 
     ***************************************************************************/
 
-    public Transaction sign (in TxType type = TxType.Payment, const(ubyte)[] data = [],
+    public Transaction sign (in OutputType outputs_type = OutputType.Payment, const(ubyte)[] data = [],
```

Comment:
@Geod24  I went with this for now to make it clearer how `OutputType` is being used.